nx.json

```diff
@@ -129,18 +129,7 @@
         }
       ],
       "options": {
-        "commands": [
-          {
-            "command": "pnpm e2e-start-local-registry"
-          },
-          {
-            "command": "pnpm e2e-build-package-publish"
-          },
-          {
-            "command": "nx run-e2e-tests {projectName}"
-          }
-        ],
-        "parallel": false
+        "commands": ["nx run-e2e-tests {projectName}"]
```

Comment:
Could we make the `targetDefaults` for `e2e` and `e2e-macos` the same as what they are for `run-e2e-tests` with these inputs? We shouldn't need a separate target after this change.

Reply:
changed